build.cfg.ts

```diff
@@ -1,7 +1,8 @@
-import { config as customConfig } from "./custom.cfg.js";
 import * as types from "./src/cfg/types";
 import { merge } from "lodash/fp";
 
+const customConfig = process.env.CUSTOM_CFG !== undefined ? JSON.parse(process?.env?.CUSTOM_CFG) : {};
```

Comment:
I like this better.

One thing to note - this config currently _will_ be built in to the build, so should not contain secrets.

Reply:
@nikonikoniko Wasn't it being baked in in the previous solution too?

Reply:
yes it was, was a general note